### PR TITLE
Update/dev portal migration

### PR DESCRIPTION
**Pull request by levfishbluefish**
Add callout that Enterprise migrations must be completed before manually migrating the Dev Portal

### CODE REVIEW

File: app/_hub/kong-inc/rate-limiting-advanced/index.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 name: Rate Limiting Advanced
 publisher: Kong Inc.
-version: 1.3-x
+version: 0.37-x
```
Comment (h3xar0n) [inline app/_hub/kong-inc/rate-limiting-advanced/index.md]:
```suggestion
version: 1.3-x
```

Comment (levfishbluefish) [reply app/_hub/kong-inc/rate-limiting-advanced/index.md]: whoops didn't mean to commit that change. Yes this fixes the dropdown bug.

**Comment by levfishbluefish** [comment]
oh no! looks like this PR got changes from #1668, I'll fix it

**Comment by levfishbluefish** [comment]
@h3xar0n  all fixed!

**Pull request merged**